boot/zephyr/mcuboot_uuid.c

```diff
@@ -0,0 +1,73 @@
+/*
+ *  Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+#include <bootutil/mcuboot_uuid.h>
+
+#define IMAGE_ID_COUNT 5
```

Comment:
as above, if this is a sample it should be moved to a sample

Reply:
I'll move it outside of the MCUboot repo too.